src/routes/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import Home from '../components/Home';
+
```

Comment:
Minor: We don't need to have empty spaces between imports, so this empty line could be deleted

Reply:
removed